scaleway/data_source_vpc_public_gateway.go

```diff
@@ -15,7 +15,7 @@ func dataSourceScalewayVPCPublicGateway() *schema.Resource {
 	dsSchema := datasourceSchemaFromResourceSchema(resourceScalewayVPCPublicGateway().Schema)
 
 	// Set 'Optional' schema elements
-	addOptionalFieldsToSchema(dsSchema, "name")
+	addOptionalFieldsToSchema(dsSchema, "name", "zone")
```

Comment:
What is the purpose of this line? zone is already in the schema of `resourceScalewayVPCPublicGateway `

Reply:
Our data-source helper, `zone` remove the optional flag. We have to precise it if we want to keep it optional. Otherwise we got this error : ```Value for unconfigurable attribute```